profiler_measurements/longDelayChannelScenario.h

```diff
@@ -0,0 +1,121 @@
+#ifndef PIPERT_LONGDELAYCHANNELSCENARIO_H
+#define PIPERT_LONGDELAYCHANNELSCENARIO_H
+#include <fcntl.h>
+#include <termio.h>
+#include <unistd.h>
+
+#include <algorithm>
+#include <iostream>
+#include <string>
+#include <thread>
+
+#include "pipert/Profiler.h"
+#include "pipert/Scheduler.h"
+#include "utils.h"
+
+/// Measurement Title : Long Delay Channel detection
+/// in this measurement we will generate three channels one of them have
+/// relatively longer processing time compared to the average processing  time
+/// of the other channels
+///
+////Expected analysis output
+/// log analyser is Expected to activate this flag for the slow channel
+// analysis logic is to to compute the average time of channels and activate this flag for the channel which is having noticable higher value
+
+namespace measurements {
+    class Human {
```

Comment:
Putting multiple classes into a single file is not wise. If you really want this, put these into an anonymous namespace which will be local to the current file.